libcipher/transpos.py

```diff
@@ -0,0 +1,35 @@
+""" 
+
+Transposition cipher implementation.
+
+Provides functions to encrypt / decrypt a message by transpose ciphering it.
+
+Encryption needs a key(a number).
+
+"""
+
+
+def encrypt(keysize,message):
+    '''
+    Args :
+        Transposition Cipher Key Size,Message
```

Comment:
Please read the Google docstring convention, mentioned in the contributing guidelines https://github.com/chennaipy/libcipher#contributing-guidelines